xbmc/pvr/windows/GUIWindowPVRBase.h

```diff
@@ -112,6 +115,12 @@ namespace PVR
   private:
     bool OpenChannelGroupSelectionDialog(void);
 
+    bool HasChannelGroupsControl();
+    bool IsChannelGroupsControlFocused();
```

Comment:
could we rename the method to `ChannelGroupsControlHasFocus`

Reply:
No. I prefer method names to start  with a verb.